core/server/common/src/test/java/alluxio/master/journal/raft/RaftJournalTest.java

```diff
@@ -563,6 +617,8 @@ private List<Integer> getFreePorts(int portCount) throws Exception {
    * Used to validate journal apply counts to master.
    */
   static class CountingDummyFileSystemMaster extends NoopMaster {
```

Comment:
Does it make sense to merge `CountintDummyFileSystemMaster` with the `CountingNoopMaster` and `SleepingNoopMaster` from `UfsJournalTest`? I feel like that can be a separate PR? If it's better included in this one then I will just merge those here.

Reply:
Yes, these can probably be refactored.

Reply:
will do in a separate one